bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/Rule.java

```diff
@@ -101,6 +104,25 @@ public String getUID() {
     }
 
     /**
+     * This method is used for getting the scope of the Rule. This property is set by the RuleEngine when
+     * the {@link Rule} is added. It's an optional property.
+     *
+     * @return scope of this {@link Rule}, an empty {@link String} if none is found
```

Comment:
You return null that is different from an empty string.
So, do you want that the caller need to check for null and non empty or does it make sense to return a non-null string all the time (empty as you written in the JavaDoc).